### PR TITLE
docs: update docs of HelperAndErrorTextTooltip, Input, Label & Link component

### PR DESCRIPTION
**Pull request by Annaleah123**
### PR Checklist

Please check if your PR fulfills the following requirements:

-   [ ] Tests for the changes have been added (for bug fixes/features)
-   [x] Docs have been added/updated (for bug fixes/features)

### PR Type

What kind of change does this PR introduce?

-   [ ] Bugfix
-   [ ] Feature
-   [ ] Code style update (formatting, local variables)
-   [ ] Refactoring (no functional changes, no api changes)
-   [ ] Build related changes
-   [ ] CI related changes
-   [ ] Performance improves
-   [ ] Adding missing tests
-   [x] Documentation content changes
-   [ ] Other... Please describe:

### Does this PR introduce a breaking change?

-   [ ] Yes
-   [x] No

### Additional context

Provide any additional context or screenshots about the feature PR.

### CODE REVIEW

File: packages/core/src/components/HelperAndErrorTextTooltip/HelperAndErrorTextTooltip.stories.mdx [hunk 1]
```diff
@@ -6,14 +6,17 @@ import { Custom } from './HelperAndErrorTextTooltip.stories.tsx';
 
 # HelperAndErrorTextTooltip
 
-HelperAndErrorTextTooltip component
+The HelperAndErrorTextTooltip displays informative text when you hover over or tap an element. When activated, it displays a text label identifying an element, such as a description of its function.
```
Comment (abhishekashyap) [inline packages/core/src/components/HelperAndErrorTextTooltip/HelperAndErrorTextTooltip.stories.mdx:9]:
```suggestion
The `HelperAndErrorTextTooltip` displays informative text when you hover over or tap an element. When activated, it displays a text label identifying an element, such as a description of its function.
```

File: packages/core/src/components/Input/Input.stories.mdx [hunk 2]
```diff
@@ -6,7 +6,9 @@ import * as stories from './Input.stories';
 
 # Input
 
-### Basic Input component with lot of customization
+The Input component allows you to get user input in a text field.
```
Comment (abhishekashyap) [inline packages/core/src/components/Input/Input.stories.mdx:9]:
```suggestion
The `Input` component allows you to get user input in a text field. 
```

File: packages/core/src/components/Label/Label.stories.mdx [hunk 3]
```diff
@@ -6,16 +6,20 @@ import * as stories from './Label.stories';
 
 # Label
 
-### Basic Label
+The Label component lets you add captions to any element in your project. You can click or tap the label to activate the associated element.
```
Comment (abhishekashyap) [inline packages/core/src/components/Label/Label.stories.mdx:9]:
```suggestion
The `Label` component lets you add captions to any element in your project. You can click or tap the label to activate the associated element.
```

File: packages/core/src/components/Link/Link.stories.mdx [hunk 4]
```diff
@@ -6,16 +6,20 @@ import * as stories from './Link.stories';
 
 # Link
 
-### Basic link component
+The Link component creates links primarily used for navigation. It semantically renders the anchor tag.
```
Comment (abhishekashyap) [inline packages/core/src/components/Link/Link.stories.mdx:9]:
```suggestion
The `Link` component creates links primarily used for navigation. It semantically renders the anchor tag.
```

**Review by abhishekashyap — CHANGES_REQUESTED** [review]

File: packages/core/src/components/HelperAndErrorTextTooltip/HelperAndErrorTextTooltip.stories.mdx [hunk 5]
```diff
@@ -6,14 +6,17 @@ import { Custom } from './HelperAndErrorTextTooltip.stories.tsx';
 
 # HelperAndErrorTextTooltip
 
-HelperAndErrorTextTooltip component
+The `HelperAndErrorTextTooltip` displays informative text when you hover over or tap an element. When activated, it displays a text label identifying an element, such as a description of its function.
```
Comment (gmukul01) [inline packages/core/src/components/HelperAndErrorTextTooltip/HelperAndErrorTextTooltip.stories.mdx:9]: We can simply keep `The HelperAndErrorTextTooltip displays informative text when you hover over or tap an element.`

File: packages/core/src/components/HelperAndErrorTextTooltip/HelperAndErrorTextTooltip.stories.mdx [hunk 6]
```diff
@@ -6,14 +6,17 @@ import { Custom } from './HelperAndErrorTextTooltip.stories.tsx';
...
+The helper prop provides feedback about an error.
```
Comment (gmukul01) [inline packages/core/src/components/HelperAndErrorTextTooltip/HelperAndErrorTextTooltip.stories.mdx:10]: There is no helper prop. Please remove this Prop

**Review by gmukul01 — CHANGES_REQUESTED** [review]

**Review by gmukul01 — APPROVED** [review]

**Pull request merged**